lsl_autobids/config_globals.py

```diff
@@ -1,6 +1,33 @@
 import os
 import yaml
 
+class CLIArgs:
+    _instance = None
+
+    def __init__(self):
+        self._args = None
+        self._defaults = {
+            "project_name": "default_project",
+            "yes": False,
```

Comment:
This is not.immideatly clear to me. I'll keep this open to check later that I understand why this exists